File: gzoom2-fe/app/src/app/api/model/workEffortAnalysisEx.ts
import { WorkEffortType } from "app/view/report-print/report";
import { WorkEffortAnalysis } from "./workEffortAnalysis";

export class WorkEffortAnalysisEx extends WorkEffortAnalysis {
    public workEffortType: WorkEffortType;
    public parentWorkEffortType: WorkEffortType;
}

File: gzoom2-be/persistence-mybatis/src/main/java/it/mapsgroup/gzoom/mybatis/dto/WorkEffortAnalysisEx.java
package it.mapsgroup.gzoom.mybatis.dto;

public class WorkEffortAnalysisEx extends WorkEffortAnalysis {
    private WorkEffortType workEffortType;
    private WorkEffortType parentWorkEffortType;

    public WorkEffortType getWorkEffortType() {
        return workEffortType;
    }

    public void setWorkEffortType(WorkEffortType workEffortType) {
        this.workEffortType = workEffortType;
    }

    public WorkEffortType getParentWorkEffortType() {
        return parentWorkEffortType;
    }

    public void setParentWorkEffortType(WorkEffortType parentWorkEffortType) {
        this.parentWorkEffortType = parentWorkEffortType;
    }
}
